addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -747,14 +740,10 @@
         <t t-if="slide.is_new_slide and not channel_progress[slide.id].get('completed')" t-call="website_slides.course_card_information"/>
         <t t-set="can_access" t-value="slide.is_preview or channel.is_member or channel.can_publish"/>
         <a t-if="can_access" t-attf-href="/slides/slide/#{slug(slide)}#{query_string}" t-title="slide.name" style="height:150px">
-            <div t-if="slide.image_1920" t-field="slide.image_1920" t-options="{'widget': 'image', 'preview_image': 'image_512'}"
-                 class="o_wslides_background_image h-100"/>
-            <div t-else="" class="o_wslides_background_image">
-                <img t-att-src="channel.website_background_image_url" class="img img-fluid"/>
-            </div>
+            <div t-field="slide.image_1920" t-options="{'widget': 'image', 'preview_image': 'image_512'}" class="o_wslides_background_image h-100"/>
         </a>
         <div t-else="" class="o_wslides_background_image" style="height:150px">
-            <img t-att-src="channel.website_background_image_url" class="img img-fluid"/>
```

Comment:
I think we should keep the getting the channel image as placeholder for the slide if the user can't access the slide.

Reply:
I added the line back.

Reply:
Nope, that does not work anymore. Please compare your runbot (https://21513559-master-all.runbot119.odoo.com/slides/trees-wood-and-gardens-3) and master (https://21512998-master-all.runbot144.odoo.com/slides/trees-wood-and-gardens-3) **as public user**

Reply:
@angv-odoo : Comment above is **not** resolved ?